### ISSUE TITLE
Victory charts support

### ISSUE THREAD

**Issue by mortocks** [EvHaus/react-pdf-charts#41]
Glad this exists. Some issues rendering with Victory charts

Example

```ts
const data = [
  { x: 1, y: 2, y0: 1 },
  { x: 2, y: 3, y0: 2 },
  { x: 3, y: 5, y0: 2 },
  { x: 4, y: 4, y0: 3 },
  { x: 5, y: 6, y0: 3 },
];

<ReactPDFChart>
  <VictoryChart>
    <VictoryBar
      data={data}
      width={100}
      height={100}
      style={{ data: { fill: "#c43a31" } }}
    />
  </VictoryChart>
</ReactPDFChart>

```

Rendering `<VictoryChart />` component by itself in web looks like this

![CleanShot 2023-08-08 at 15 12 22](https://github.com/EvHaus/react-pdf-charts/assets/314196/e55cc9db-8793-4eb7-b196-7d05cb35bc01)

but in `<ReactPFDChart/>` / `react-pdf` it doesn't render the bars. 

![CleanShot 2023-08-08 at 15 13 06](https://github.com/EvHaus/react-pdf-charts/assets/314196/068a4f84-a01f-4066-b63c-f8ebeaef48d0)

Might just be a color issue (white on white) or the bars aren't rendering at all

**Comment by EvHaus** [comment]
Thanks for report! I was planning on getting Victory support working but haven't had a chance to yet. 

Looking at it briefly now, it seems that when you render a Victory chart with `ReactDOMServer.renderToStaticMarkup` -- it renders an empty `<svg/>` element for the bars... I'm not sure why that's happening. Might be something in Victory itself that blocks server-side rendering?

**Comment by EvHaus** [comment]
Thanks for the report! I was meaning to add support for Victory. I took a quick look and you are correct -- the styling wasn't being applied to the bars.

I just published v0.1.0 which includes some basic support for this. The result is a lot closer to your web-based image:

![image](https://github.com/EvHaus/react-pdf-charts/assets/226640/839ade09-97bb-4d6f-9761-f312cd7a5425)
